### PR TITLE
Fixing wrong links

### PR DESCRIPTION
**Pull request by l0k0ms**
Removes the localisation over ressources file

### CODE REVIEW

File: content/fr/integrations/amazon_ecs.md [hunk 1]
```diff
@@ -292,44 +292,37 @@ Pour réduire les données parasites, l'intégration AWS ECS est automatiquemen
...
-
-
-{{< get-dependencies >}}
\ No newline at end of file
+[4]: https://docs.datadoghq.com/fr/tracing/setup/docker
+[5]: https://aws.amazon.com/cli
+[6]: /json/datadog-agent-ecs.json
+[7]: /json/datadog-agent-ecs1.json
+[8]: https://app.datadoghq.com/account/settings#api
+[9]:
```
Comment (ruthnaebeck) [inline content/fr/integrations/amazon_ecs.md]: empty link?

Comment (l0k0ms) [inline content/fr/integrations/amazon_ecs.md]:
```suggestion
[9]: #collecte-de-processus
```

File: content/fr/integrations/amazon_ecs.md [hunk 2]
```diff
@@ -292,44 +292,37 @@ Pour réduire les données parasites, l'intégration AWS ECS est automatiquemen
...
-
-
-{{< get-dependencies >}}
\ No newline at end of file
+[4]: https://docs.datadoghq.com/fr/tracing/setup/docker
+[5]: https://aws.amazon.com/cli
+[6]: /json/datadog-agent-ecs.json
+[7]: /json/datadog-agent-ecs1.json
+[8]: https://app.datadoghq.com/account/settings#api
+[9]: #collecte-de-processus
```
Comment (ruthnaebeck) [inline content/fr/integrations/amazon_ecs.md]: Since this is on the same page, shouldn't it be linked above?

Comment (l0k0ms) [reply content/fr/integrations/amazon_ecs.md]: we could but it's not mandatory since it's a translated files and it might be override soon

**Pull request merged**